index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="toolbar">
+      <div class="logo_placeholder">
+        <a href="index.html" class="logo">Welcome</a>
+      </div>
+      <nav class="menu">
+        <button type="button" class="btn">
+          <i class="material-icons">menu</i>
+        </button>
+      </nav>
+    </header>
+    <section class="headline background">
+      <div>
+        <h1>Hey there. I’m Milan</h1>
+        <h2>I’m a software developer</h2>
+      </div>
+      <p>
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <a href="#"
+          ><img class="icon" src="icons/twitter.svg" alt="twitter logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/linkedin.svg" alt="linkedin logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/medium.svg" alt="medium logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/github.svg" alt="github logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/angellist.svg" alt="angellist logo"
+        /></a>
+      </div>
+      <button type="button" class="btn expand">
+        <i class="material-icons">expand_more</i>
+      </button>
+    </section>
+    <section class="recent_work section">
+      <h1>My Recent Works</h1>
+      <div class="indicator">&nbsp;</div>
+
+      <div class="work_cards">
+        <article class="card">
+          <div class="card_img">&nbsp;</div>
+          <div class="card_details abs bottom_0">
+            <h1>
+              Multi-Post Stories <br />
+              Gain+Glory
+            </h1>
+
+            <ul class="tags">
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">Ruby on rails</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">CSS</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>

```

Comment:
- [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.